R/load_flu_hosp_data.R

```diff
@@ -142,6 +163,37 @@ load_flu_hosp_data <- function(as_of = NULL,
     # drop data for locations retrieved from covidcast,
     # but not included in forecasting exercise -- mainly American Samoa
     dplyr::filter(!is.na(location))
-  
+
   return(final_data)
-}
\ No newline at end of file
+}
+
+
+#' Retrieving flu data
+#'
+#' @param as_of Date or string in format "YYYY-MM-DD" specifying the date which
+#'   the data was available on or before. If `NULL`, the default returns the
+#'   most recent available data.
+#' @param locations optional list of FIPS or location abbreviations. Defaults to
+#'   the US, the 50 states, DC, PR, and VI.
+#' @param temporal_resolution "daily" or "weekly"
+#' @param source either "covidcast" or "HealthData". HealthData only supports
+#' `as_of` being `NULL` or the current date.
+#' @param na.rm boolean indicating whether NA values should be dropped when
+#'   aggregating state-level values and calculating weekly totals. Defaults to
+#'   `FALSE`
+#'
+#' @return data frame of flu incidence with columns date, location,
+#'   location_name, value
+load_flu_hosp_data <- function(as_of = NULL,
+                               locations = "*",
+                               temporal_resolution = "daily",
+                               source = "HealthData",
+                               na.rm = FALSE) {
+  load_hosp_data(pathogen = "flu",
+                 as_of = as_of,
+                 temporal_resolution = temporal_resolution,
```

Comment:
Think we need to add locations parameter in here
```suggestion
                 locations = locations,
                 temporal_resolution = temporal_resolution,
```

Reply:
I'll add this too.